cypress/e2e/pages/forgotPassword.page.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types = "cypress" />
+
+const baseUrl = 'http://localhost:4200/auth/request-password'
```

Comment:
```suggestion
const pageUrl = '/auth/request-password'
```
baseUrl вказаний в сайпрес конфіг